### ISSUE TITLE
[Other issue]: Double check that .github/ISSUE_TEMPLATE YAMLs are valid

### ISSUE THREAD

**Issue by nickvisut** [sfbrigade/support-sfusd#200]
### What is the issue you are reporting?

Redhat YAML validator (VSCode extension) is reporting issues; double check if these are valid, or if they're false positives.

**Comment by mattgianni** [comment]
I reviewed the GitHub docs on this topic, and it seems like `config.yml` is correct. The validator is whining about it, but I believe it's using the wrong schema to evaluate this config file. Our `config.yml` looks nearly identical to the documentation, so I think the validator must be wrong. More importantly, our version is working as expected.

The validator is also flagging `other.yml` for the empty string in the `lablels` array. Here's the offending line:

```yaml
labels: [""]
```

I tried removing this line completely, and everything continued to work just fine. This empty string doesn't seem to **hurt** anything, but it doesn't seem necessary either.

Also, since we are *not* accepting blank issues (ie `blank_issues_enabled: false`), according to the [docs](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) this means that the default `ISSUE_TEMPLATE.md` file should never be used. So it should be safe to remove it.
